src/convertToObject.js

```diff
@@ -3,16 +3,34 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arrWithStyles = sourceString
+    .split(';')
+    .map(str => str.trim())
+    .filter(str => str.length > 1);
+
+  const objWithStyles = arrWithStyles.reduce((prev, str) => {
+    const arrWithValues = str.split(':');
+    const key = arrWithValues[0].trim();
+    const value = arrWithValues[1].trim();
```

Comment:
don't create this variables inside reduce method, otherwise on each call of this method they will create over and over again